tests/ui/traits/next-solver/deep-canonical.rs

```diff
@@ -0,0 +1,84 @@
+//@ check-pass
+//@ compile-flags: -Znext-solver
```

Comment:
instead of adding a new test, please just add revisions to `tests/ui/associated-consts/issue-93775.rs`

Reply:
oh lol i didn't realize this was minimized from an existing test 👍